src/components/MenuPanel/SwipeContainer.js

```diff
@@ -45,6 +42,10 @@ class SwipeContainer extends React.Component {
           initial: [xInitial],
           xy: [x],
         }) => {
+          if (!enabled) {
```

Comment:
What was your reasoning for moving this here?

Reply:
This is to keep the same node hierarchy: this way, `Container` and its children tree doesn’t get unmounted / remounted. It solves the issue of having the current app reloading when the viewport is resized.